src/EmployeeController.java

```diff
@@ -0,0 +1,86 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class EmployeeController {
+    public static void main(String args[]) {
+        HashMap<String,Integer> org=new HashMap<String,Integer>();
+        org.put("Robosoft",1);
+        org.put("Infosys",2);
+        org.put("MindTree",3);
+
+
+        Employee e1=new Employee("Raj",empNum(),"12/06/1999","bengaluru",Designation.SYSTEM_ENGINEER,org.get("Robosoft"));
+        Employee e2=new Employee("Adam",empNum(),"19/08/1999","bengaluru",Designation.TRAINEE,org.get("Infosys"));
+        Employee e3=new Employee("Seema",empNum(),"12/07/1966","Udupi",Designation.PROJECT_LEADER,org.get("MindTree"));
+        Employee e4=new Employee("Ameer",empNum(),"23/06/1976","Manglore",Designation.SYSTEM_ENGINEER,org.get("Robosoft"));
+        Employee e5=new Employee("Sheeba",empNum(),"23/07/1997","bengaluru",Designation.SYSTEM_ENGINEER,org.get("MindTree"));
+        EmployeeService es=new EmployeeService();
+        es.register(e1);
+        es.register(e2);
+        es.register(e3);
+        es.register(e4);
+        es.register(e5);
+        System.out.println("Employee list before sorting");
+        Employee e6=new Employee();
```

Comment:
Very nicely done. 
Suggestion: 
can you write a separate method to create employee object. Pass all required value to it as input parameter and use that method to create Employee object ?